src/BoxList.php

```diff
@@ -59,11 +59,11 @@ public static function compare($boxA, $boxB): int
         $boxBVolume = $boxB->getInnerWidth() * $boxB->getInnerLength() * $boxB->getInnerDepth();
 
         $volumeDecider = $boxAVolume <=> $boxBVolume; // try smallest box first
-        $emptyWeightDecider = $boxA->getEmptyWeight() <=> $boxB->getEmptyWeight(); // with smallest empty weight
-
         if ($volumeDecider !== 0) {
             return $volumeDecider;
         }
+
+        $emptyWeightDecider = $boxA->getEmptyWeight() <=> $boxB->getEmptyWeight(); // with smallest empty weight
```

Comment:
Good spot, there's definitely an optimisation to be had here. But also because of this, it's not possible to tell from looking at the timings on the Travis run what the performance impact of the other change is.

So what I'm going to do is steal this bit out of the PR and commit it to master. Then the comparison for the rest should be more straightforward.

Reply:
Sounds good. Just FYI I'm testing with ~170 boxes so while the unit tests might show a small improvement I think the improvement for cases like mine will be much more significant. I don't have solid numbers yet, was just looking for some low hanging fruit.

Reply:
170! 😱 

I love a good torture test, it would be great to have a sample of that in the test suite if that would be OK? (`tests/PackerTest.php` would be the right placed to put it). Just a single example of the typical items in a packing and the 170 boxes.

Apart from obviously helping evaluate this PR in particular, it would also ensure that performance on that kind of scenario stays good going forward and doesn't regress.

Reply:
I'll work on adding such a test. Any other thoughts in this PR as it is written currently?